### PR TITLE
add lightgbm demo

### PR DESCRIPTION
**Pull request by partrita**
I just made simple code for ML.
Thanks in advance.

### CODE REVIEW

File: examples/lightgbm/pixi.toml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+[project]
+name = "lightgbm"
+version = "0.1.0"
+description = "LightGBM demo for machine learning project"
+authors = ["fkt <deepthought@postech.ac.kr>"]
+channels = ["conda-forge"]
+platforms = ["linux-64"]
```
Comment (ruben-arts) [inline examples/lightgbm/pixi.toml:7]:
```suggestion
platforms = ["linux-64", "osx-arm64", "osx-64", "win-64"]
```
We always try to make the examples as multi platform as possible. We can test it on all systems. Adding this does require to rerender the `pixi.lock` Could you do that?

File: examples/lightgbm/pixi.toml [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+[tasks]
```
Comment (ruben-arts) [inline examples/lightgbm/pixi.toml:9]:
```suggestion
[tasks]
start = "python main.py"
```
We try to always add a `start` command :smile:

File: examples/lightgbm/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+This simple code to build and evaluate a LightGBM classification model using the Wisconsin Breast Cancer dataset.
+
+
+## Usage
+
+1. Check dataset: `breast_cancer_data.csv`
+2. Run the code:
+
+```bash
+pixi run python main.py
```
Comment (ruben-arts) [inline examples/lightgbm/README.md:12]:
```suggestion
pixi run start
```
After the change in the `pixi.toml`

**Review by ruben-arts — CHANGES_REQUESTED** [review]
Awesome @partrita Thank you!
Left a few small requests.

**Comment by ruben-arts** [comment]
He @partrita This looks good, could you also commit the updated the `pixi.lock`. I you don't have it yet you can run `pixi install` and then push the changes!

**Comment by partrita** [comment]
> He @partrita This looks good, could you also commit the updated the `pixi.lock`. I you don't have it yet you can run `pixi install` and then push the changes!

Ok, I just commited. Thanks!

**Review by ruben-arts — APPROVED** [review]

**Comment by ruben-arts** [comment]
Sorry, I forgot to run start CI on your PR. You should be able to test this locally with `pixi run lint`.

**Comment by partrita** [comment]
> Sorry, I forgot to run start CI on your PR. You should be able to test this locally with `pixi run lint`.

`pixi run lint` command was not working, I do this.

```bash
pixi global install pre-commit
pre-commit install //in project folder
git add .
git commit -m ""
git push
```

Thanks!

**Pull request merged**